docs/user/cassandra_datastore.rst

```diff
@@ -46,15 +46,15 @@ installation directory).
 
 After you've successfully installed and configured Cassandra, you can
 move on to installing the GeoMesa Cassandra distribution.
-Start by choosing or creating a directory where you want
-to put it. Then ``cd`` into that directory and type::
+To start, go to
+https://repo.locationtech.org/content/repositories/geomesa-snapshots/org/locationtech/geomesa/geomesa-cassandra-dist_2.11/1.3.0-m3-SNAPSHOT/ ,
```

Comment:
sounds good - @mdzimmerman let's make sure we get the changes in the 1.3.0 docs.